pjnath/src/pjnath/ice_session.c

```diff
@@ -3141,6 +3144,13 @@ static pj_status_t on_stun_rx_request(pj_stun_session *sess,
     uc_attr = (pj_stun_use_candidate_attr*)
               pj_stun_msg_find_attr(msg, PJ_STUN_ATTR_USE_CANDIDATE, 0);
 
+    if (uc_attr != NULL && ice->is_complete) {
+        LOG4((ice->obj_name,
+            "Ignored incoming check after ICE nego has been completed!, "
+            "no need to send response"));
+        pj_grp_lock_release(ice->grp_lock);
+        return PJ_SUCCESS;
+    }
```

Comment:
This part seems rather OOT and change the ICE flow/algo? Usually this kind of changes is specified (including the reason) in the PR desc.